tasks/src/document-question-answering/data.ts

```diff
@@ -0,0 +1,57 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from the 2020 DocVQA challenge. The documents are taken from the UCSF Industry Documents Library.",
+			id:          "eliolio/docvqa",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Question",
+				content: "Which is the most critical issue for live rep support?",
+				type:    "text",
+			},
+			{
+				filename: "document-question-answering-input.jpeg",
```

Comment:
Maybe we could have a smaller document that people can see given schema is small 🙂

Reply:
I updated the document image for a cropped version